ci/matrix.yml

```diff
@@ -41,6 +42,6 @@ devcontainer_version: '23.08'
 # Configurations that will run for every PR
 pull_request:
   nvcc:
-    # There is currently only one CUDA 11.8 image available which comes with the system's default C++ compiler. For ubuntu22.04, we know that the default CC is gcc11.3
-    - {cuda: *cuda_oldest, os: 'ubuntu22.04', cpu: 'amd64', compiler: {name: 'gcc', version: '11', exe: 'c++'}, gpu_build_archs: '60', std: [17], jobs: ['build', 'test']}
-    - {cuda: *cuda_newest, os: 'ubuntu22.04', cpu: 'amd64', compiler: {name: 'gcc', version: '12', exe: 'g++'}, gpu_build_archs: '70', std: [17], jobs: ['build', 'test']}
\ No newline at end of file
+    - {cuda: *cuda_oldest, os: 'ubuntu22.04', cpu: 'amd64', compiler: {name: 'gcc', version: '11', exe: 'g++'}, gpu_build_archs: '70', std: [17], jobs: ['build', 'test']}
+    - {cuda: *cuda_newest, os: 'ubuntu22.04', cpu: 'amd64', compiler: {name: 'gcc', version: '12', exe: 'g++'}, gpu_build_archs: '70', std: [17], jobs: ['build', 'test']}
+    - {cuda: *cuda_newest, os: 'ubuntu22.04', cpu: 'arm64', compiler: {name: 'gcc', version: '12', exe: 'g++'}, gpu_build_archs: '60,90', std: [17], jobs: ['build']}
```

Comment:
Can we also test this?

Reply:
I think we don't have arm64 GPU runners atm but let me check.

Reply:
Yep, no arm64 gpu runners yet